### PR TITLE
Fix BS3 regressions

### LINKED ISSUE #6063: Bootstrap 3 regressions
Let's collect the regressions introduced by the Bootstrap 3 port in this issue.

* [ ] Scroll bar for "to" input in conversations view (Chromium 43.0.2357.65) screenshots: [[1]](https://cloud.githubusercontent.com/assets/3798614/7995910/f632ffde-0b1a-11e5-8b6d-85bfb93ab990.png)
  [[2]](https://cloud.githubusercontent.com/assets/3798614/7995909/f630124c-0b1a-11e5-80c0-f8a41644544c.png)
* [ ] Avatars in conversation view hidden (Chromium 43.0.2357.65) [screenshot](https://cloud.githubusercontent.com/assets/3798614/7995927/3bf331b0-0b1b-11e5-9ba7-ca94f665dae6.png)
* [x] The dropdown aspect and the "mark as read" icon are too close to the right of the screen on the notification page. ([comment](https://github.com/diaspora/diaspora/issues/6063#issuecomment-109179380)) ([commit](https://github.com/diaspora/diaspora/commit/7b7700b31722371118b0bfa2eb8c2314b56796a4))
* [x] The dropdown to add a tag in the left side bar of the stream is misplaced (gap between it and the input) ([comment](https://github.com/diaspora/diaspora/issues/6063#issuecomment-109179380)) ([commit](https://github.com/diaspora/diaspora/commit/7b7700b31722371118b0bfa2eb8c2314b56796a4))
* [x] Avatars of people who likes are collapsed in the stream view ([comment](https://github.com/diaspora/diaspora/issues/6063#issuecomment-109180187)) ([commit](https://github.com/diaspora/diaspora/commit/7b7700b31722371118b0bfa2eb8c2314b56796a4))
* [x] Images post by link are cropped ([comment](https://github.com/diaspora/diaspora/issues/6063#issuecomment-109210968)) ([commit](https://github.com/diaspora/diaspora/commit/39fe6bfff231458aca2f4b075e870675bc369c3f))
* [x] No more animation when uploading an image ([comment](https://github.com/diaspora/diaspora/issues/6063#issuecomment-109210968))
* [x] Stream view and notifications view are not as wide as they were on mobile view. ([comment](https://github.com/diaspora/diaspora/issues/6063#issuecomment-109212095))
* [x] notifications content lacks padding (mobile) ([comment](https://github.com/diaspora/diaspora/issues/6063#issuecomment-109212095))
* [x] Cancel button of "Change password" lacks margin on mobile view for small screens ([comment](https://github.com/diaspora/diaspora/issues/6063#issuecomment-109212482))
* [x] Avatar lacks left margin on on contacts page for mobile view. ([comment](https://github.com/diaspora/diaspora/issues/6063#issuecomment-109212748))
* [x] Player button for oEmbed content has not the right height ([comment](https://github.com/diaspora/diaspora/issues/6063#issuecomment-109354617))
* [x] SPV for reshared posts is broken ([comment](https://github.com/diaspora/diaspora/issues/6063#issuecomment-109450895))
* [x] loading spinner broken in streams ([comment](https://github.com/diaspora/diaspora/issues/6063#issuecomment-109581123))
* [x] overflow on #tag pages #6142 
* [x] loading spinner for location input in publisher is too big
* [x] Formatting of ignored users in settings -> privacy has broken - used to be one user per line, now they just pile up one after another in the same paragraph. #6349
* [ ] Mobile bookmarklet service icons opacity is the same whether clicked or not ([comment](https://github.com/diaspora/diaspora/issues/6063#issuecomment-128135786))
* [x] Photo viewer doesn't scale photo to screen size ([comment](https://github.com/diaspora/diaspora/issues/6063#issuecomment-128142946)) #6301
* [ ] On the settings page we have a `.container-fluid` div inside of a `.container-fluid` div
* [x] Commenting on stream posts on mobile causes comment not to be loaded properly ([comment and screenshots](https://github.com/diaspora/diaspora/issues/6063#issuecomment-139368965))

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/20517292-bootstrap-3-regressions?utm_campaign=plugin&utm_content=tracker%2F522&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F522&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by svbergerem**
Fixes #6063.

### CODE REVIEW

File: app/assets/stylesheets/conversations.scss [hunk 1]
```diff
@@ -1,6 +1,10 @@
 #conversations_container {
   .stream_container { border-left: none; }
 
+  .stream_element:not(.conversation) .avatar {
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/conversations.scss]: Selector `stream_element` should be written in lowercase with hyphens

File: app/assets/stylesheets/mobile/mobile.scss [hunk 2]
```diff
@@ -406,15 +406,11 @@ h3 {  margin-top: 0; }
 #new_status_message {
   margin: 0;
 
-  fieldset {
-    padding: 10px;
-
-    .service_icon {
-      cursor: pointer;
+  .service_icon {
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/mobile/mobile.scss]: Selector `service_icon` should be written in lowercase with hyphens

File: app/assets/stylesheets/conversations.scss (hunk 1 above)
Comment (svbergerem) [inline app/assets/stylesheets/conversations.scss]: Out of scope of this PR.

```bash
$ grep -Rn "stream_element" app features spec | wc
155
```

File: app/assets/stylesheets/mobile/mobile.scss (hunk 2 above)
Comment (svbergerem) [inline app/assets/stylesheets/mobile/mobile.scss]: Out of scope of this PR.

```bash
$ grep -Rn "service_icon" app features spec | wc
25
```

**Comment by jhass** [comment]
Thank you!

**Pull request merged**